src/Action/LookupAction.php

```diff
@@ -56,7 +56,8 @@ protected function _getFindConfig()
         $request = $this->_request();
 
         $config = (array)$this->config('findConfig');
-        if ($idField = $request->query('id')) {
+
+        if (($idField = $request->query('id')) && in_array($idField, $this->_table()->schema()->columns())) {
```

Comment:
Agree I also don't like it. I will adjust it. Additionally I would do the same conditions for `valueField`, what do you think?